packages/components/psammead-caption/src/__snapshots__/index.test.jsx.snap

```diff
@@ -89,8 +90,9 @@ exports[`Caption should render with some offscreen text and arabic script typogr
   font-size: 1.0625rem;
   line-height: 1.5rem;
   color: #6E6E73;
-  font-family: ReithSans,Helvetica,Arial,sans-serif;
-  font-style: italic;
+  font-family: NassimPersian,Arial,Verdana,Geneva,Helvetica,sans serif;
+  font-weight: 400;
+  font-style: normal;
```

Comment:
Just starting review, but is this right? we replace italic with normal

Reply:
This is for `persian` which doesn't have italic, so its falling back to `normal`

Reply:
Captions have some specific requirements re text style, could @sareh @eleanombre have look at this PR please and see if you spot anything?